src/TouristInfoCards.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+import CityCard from "./CityCard.js";
+
+function TouristInfoCards(props) {
+  const card = (
+    <div className="Tourist-Cards">
+      {props.cities.map(city => {
+        return <CityCard key={city.name} city={city} />;
```

Comment:
Love that you've abstracted the city card stuff into a separate component `CityCard`